lib/NativeCode/DynamicLibraryLoaderHelper/NativeRender/src/dllmain.cpp

```diff
@@ -125,21 +126,15 @@ void get_cli_arguments(config_legacy::EOSConfig eos_config)
 #if PLATFORM_32BITS
 #pragma comment(linker, "/export:UnityPluginLoad=_UnityPluginLoad@4")
 #endif
-PEW_EOS_API_FUNC(void) UnityPluginLoad(void*)
+PEW_EOS_API_FUNC(void) UnityPluginLoad(void* arg)
 {
+    // TODO: Only display this dialog when not running in debug from within
+    //       Visual Studio.
```

Comment:
Remove comment - since this has been accomplished, as is clear from the changes below.